website/docs/r/windows_web_app.html.markdown

```diff
@@ -0,0 +1,591 @@
+
+* `path` - (Optional) The path to which this rule status code applies. 
+
+* `sub_status` - (Optional) The Request Sub Status of the Status Code.
+
+* `win32_status` - (Optional) The Win32 Status Code of the Request.
+
+---
+
+A `storage_account` block supports the following:
+
+* `access_key` - (Required) The Access key for the storage account.
+
+* `account_name` - (Required) The Name of the Storage Account.
+
+* `name` - (Required) The name which should be used for this TODO.
+
+* `share_name` - (Required) The Name of the File Share or Container Name for Blob storage.
+
+* `type` - (Required) The Azure Storage Type. Possible values include `AzureFiles` and `AzureBlob`
+
+* `mount_path` - (Optional) The path at which to mount the storage share.
+
+---
+
+A `trigger` block supports the following:
+
+* `private_memory_kb` - (Optional) The amount of Private Memory to be consumed for this rule to trigger. Possible values are between `102400` and  `13631488`.
+
+* `requests` - (Optional) A `requests` block as defined above.
+
+* `slow_request` - (Optional) One or more `slow_request` blocks as defined above.
+
+* `status_code` - (Optional) One or more `status_code` blocks as defined above.
+
+---
+
+A `twitter` block supports the following:
+
+* `consumer_key` - (Required) The OAuth 1.0a consumer key of the Twitter application used for sign-in. 
+  
+* `consumer_secret` - (Optional) The OAuth 1.0a consumer secret of the Twitter application used for sign-in. Cannot be specified with `consumer_secret_setting_name`.
+  
+* `consumer_secret_setting_name` - (Optional) The app setting name that contains the OAuth 1.0a consumer secret of the Twitter application used for sign-in. Cannot be specified with `consumer_secret`.
+
+---
+
+A `virtual_application` block supports the following:
+
+* `physical_path` - (Required) The physical path for the Virtual Application.
+
+* `preload` - (Required) Should pre-loading be enabled. Defaults to `false`.
+
+* `virtual_directory` - (Optional) One or more `virtual_directory` blocks as defined below.
+
+* `virtual_path` - (Required) The Virtual Path for the Virtual Application.
+
+---
+
+A `virtual_directory` block supports the following:
+
+* `physical_path` - (Optional) The physical path for the Virtual Application.
+
+* `virtual_path` - (Optional) The Virtual Path for the Virtual Application.
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported: 
+
+* `id` - The ID of the Windows Web App.
+
+* `custom_domain_verification_id` - The ID of the TODO.
+
+* `default_hostname` - The default hostname of the Windows Web App.
+
+* `kind` - The Kind value for this Windows Web App. 
+
+* `outbound_ip_address_list` - A list of outbound IP addresses - such as `["52.23.25.3", "52.143.43.12"]`
+
+* `outbound_ip_addresses` - A comma separated list of outbound IP addresses - such as `52.23.25.3,52.143.43.12`.
```

Comment:
what happens if you specify both of these? are they the same thing?

Reply:
These are outputs/ `Computed` only.